changelog-tool/changelog.pony

```diff
@@ -79,25 +84,25 @@ class Release
     added = Section._empty(Added)
     changed = Section._empty(Changed)
 
+  fun ref add_entry(section_name: String, entry: String) ? =>
+    let section =
+      match section_name
+      | "fixed" => try fixed as Section else fixed = Section._empty(Fixed); fixed as Section end
```

Comment:
These lines also extend past the 80 column limit.

Reply:
So, took the opportunity to use the `if` instead of `try` just like your example in the zulip conversation.